docs/source/user/setup.rst

```diff
@@ -268,7 +268,7 @@ You can also bulk insert a set of stations from a CSV file, by defining the stat
 .. code-block:: bash
 
    python3 wis2box-ctl.py login
-   wis2box metadata station publish-collection
+   wis2box metadata station publish-collection /data/wis2box/metadata/stations/station_list.csv
 
 After doing a bulk insert please review the stations in wis2box-webapp and associate each station to the correct topics.
```

Comment:
Can you please update the documentation to include the topic-association during the bulk-insert ? I am not sure what is the use-case for having station metadata in wis2box for stations that are not associated to WIS2-topic to be published, so adding a topic during the bulk-insert should be the baseline.